### PR TITLE
List Implementation - E remove(int index)

### LINKED ISSUE #22: Implement : List Interface - E remove(int index)

### PR DESCRIPTION
**Pull request by vatsalkesarwani12**
#22 Implement : List Interface - E remove(int index)

### CODE REVIEW

**Review by vatsalkesarwani12 — COMMENTED** [review]
#21 Implement : List Interface - int indexOf(Object o)

**Comment by jyotijauhari** [comment]
@plazzy99  send 2 different pr with the issue name mentioned in the issue.
(also add issue number in description of pr)

**Comment by vatsalkesarwani12** [comment]
@JyotiJauhari deleted file 
added one pr
please check

File: Collection Interfaces/List Interface/Solution.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import java.util.*;
```
Comment (jyotijauhari) [inline Collection Interfaces/List Interface/Solution.java]: (in line 2-3)
add documentation(in form of comment explaining [ E remove(int index) method ] in 2-3 lines.
refer :  [this ](https://github.com/VBSquad/Scrabbling-Java-With-CF/blob/master/Collection%20Interfaces/List%20Interface/boolean%20isEmpty()/isEmptyFunc.java)

File: Collection Interfaces/List Interface/Solution.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+public class boardMoves{
+    //main code
+    public static void main (String[] args)
```
Comment (jyotijauhari) [inline Collection Interfaces/List Interface/Solution.java]: Please add space around operators at all places
also space aound (
space around [
space around <

File: Collection Interfaces/List Interface/Solution.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  
+        // remove element 
+        list.remove(index); 
+  
+        // Final list 
+        System.out.println("Final List: " + list); 
+        
+	sc.close();
+    }
+}
```
Comment (jyotijauhari) [inline Collection Interfaces/List Interface/Solution.java]: add sample input and output in form of comments at last  .

File: Collection Interfaces/List Interface/Solution.java (hunk 2 above)
Comment (vatsalkesarwani12) [reply Collection Interfaces/List Interface/Solution.java]: ok

File: Collection Interfaces/List Interface/Solution.java (hunk 1 above)
Comment (jyotijauhari) [reply Collection Interfaces/List Interface/Solution.java]: for now add this : The remove(int index) method of List interface in Java is used to remove an element from the specified index from a List container and returns the element after removing it. It also shifts the elements after the removed element by 1 position to the left in the List.

**Review by vatsalkesarwani12 — COMMENTED** [review]
@JyotiJauhari changes updated.

**Comment by jyotijauhari** [comment]
@plazzy99 change file name to method name e remove (int item).java if possible.

**Comment by vatsalkesarwani12** [comment]
@JyotiJauhari changes updated , thank you for so much cooperation.

**Comment by Dhruv-194** [comment]
@plazzy99 There are two more changes which I need you to do - 

- The code has a syntax error at line 13  `Scanner sc=new Scanner();` in this code. There is no suitable constructor found for Scanner(no arguments). 

- Moreover add this java file in a folder with the folder name set to this method's name, and then finally add this folder with the java file in the List Interface Folder.  ( for reference see the project and the merged PR) #46

**Pull request merged**